invenio_app_ils/documents/facets.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""ILS Document indexer APIs."""
+
+import re
+
+from elasticsearch_dsl import Q
+from flask import current_app
+from invenio_rest.errors import FieldError, RESTValidationError
+
+
+def custom_metadata_filter(field):
```

Comment:
is this to be removed after our chat?  
I am sorry if you spent a lot of time on this code :(